app/components/popover-link-component.coffee

```diff
@@ -10,14 +13,6 @@ Ember.Widgets.PopoverLinkComponent = Ember.Component.extend
   rootElement: '.ember-application'
   fade: yes
 
-  willDestroy: ->
```

Comment:
yeah, my 'rebase' method right now is basically to to `--ours` and then manually pull in the changes